plugin/pkg/admission/podnodeselector/admission.go

```diff
@@ -259,3 +259,20 @@ func (p *Plugin) getNodeSelectorMap(namespace *corev1.Namespace) (labels.Set, er
 	}
 	return selector, nil
 }
+
+func isSubset(subSet, superSet labels.Set) bool {
+	if len(superSet) == 0 {
+		return false
```

Comment:
Please don't change the logic from the original.  This method isn't for re-use, it's for whatever this admission plugin needs.  Looks like the intent is for an empty list to always allow.

Reply:
thanks, I understand it,  the logic should not change. It's my mistake.